defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+#openstack_undercloud_version: 13
```

Comment:
Why is this commented out?

Reply:
It's a variable that is used by the role, but I didn't know how I feel about making a default for it. Or, if so, what to set the value to.

Reply:
I don't think it's unreasonable to set this to the latest released version.

Reply:
13 is the latest LTS, isn't it? Do we want to track every version or only LTS versions?

Reply:
I think latest LTS is fine.  We can always change this later depending on requirements.